cypress/integration/models/stakeholders.ts

```diff
@@ -54,16 +89,21 @@ export class Stakeholders {
             });
         this.getStakeholderName();
         this.fillName(this.stakeholderName);
-        // jobfunction and group edit to be implemented
+        if (jobfunction) {
+            this.selectJobfunction(jobfunction);
+        }
+        if (groups) {
+            this.removeGroup(groups[0]);
```

Comment:
@sshveta We can do that. In this current approach, actually we have control of what groups we are providing in the test file. I am creating stakeholder with 2 groups and in edit, removing one of them. 
Can we take like use both things? Update and remove, since there is already a bug for remove not working, I want to make sure, we test it every time when our tests run

Reply:
Ok yea we can do that . 
The current edit method takes a group name and removes it .
 How will you test adding one more group using this approach ? let's say you have one group and you want to add one more group ? or you want to remove one and add another one . Will this method do that too ?

Reply:
If not then we can selectmember() instead of addgroup or removegroup().

Selectgroup will add group if not already added and remove group if already added .Right ?

Reply:
We can do it, with current method, as it accepts an array. Will make the changes that covers your point.

Reply:
Updated with changes, instead of using add and remove methods, now using only one single edit method with selectGroups() functions